API.md

```diff
@@ -0,0 +1,123 @@
+# Authmosphere API documentation
+
+## Mock tooling
+
+This tooling provides an abstraction to easily mock oAuth 2.0 (RFC) related endpoints.
```

Comment:
We should use `OAuth` instead `oAuth` consistently I think.

Reply:
RFC should be a link I guess?

Reply:
✅